src/components/_progress.scss

```diff
@@ -0,0 +1,52 @@
+.progress {
+  @include border-style(5);
+  border: 2px solid $primary;
+  box-shadow: 2px 8px 4px -6px hsla(0%, 0%, 0%, 0.3);
```

Comment:
In `src/core/_config.scss` we have defined sizes for shadows, and the `shadow()` mixin. 
https://github.com/papercss/papercss/blob/b6db8fad0659f45e04cf45f15b7ba96409c5a2bb/src/core/_config.scss#L80
This comes pretty close to what you have defined here. I'd rathe use the predefined values instead of adding new ones. Try trplacing this with the values defined in `config`.